lib/constable/models/user.ex

```diff
@@ -72,9 +72,15 @@ defmodule Constable.User do
     changeset
     |> validate_change(:email, fn :email, value ->
       case String.split(value, "@") do
-        [_, @permitted_email_domain] -> []
-        _ -> [email: "must be a member of #{@permitted_email_domain}"]
-      end
+        [_, domain] ->
```

Comment:
you can't use a function in the pattern match here, which is why I was reaching for something other than the original line

Reply:
What do you think about using [`String.ends_with?/2`](https://hexdocs.pm/elixir/String.html#ends_with?/2) here instead of doing an `if` statement within the `case` statement? 

```elixir
if String.ends_with?(value, "@#{permitted_email_domain()}") do
  []
else
  [email: "must be a member of #{permitted_email_domain()}"]
end
```